libManager/libManager.go

```diff
@@ -0,0 +1,133 @@
+package libManager
+
+import (
+	"path/filepath"
+
+	"github.com/future-architect/vuls/models"
+	"github.com/future-architect/vuls/util"
+	godeptypes "github.com/knqyf263/go-dep-parser/pkg/types"
+	"github.com/knqyf263/go-version"
+	"github.com/knqyf263/trivy/pkg/db"
+	"github.com/knqyf263/trivy/pkg/log"
+	"github.com/knqyf263/trivy/pkg/scanner/library"
+	"github.com/knqyf263/trivy/pkg/vulnsrc/vulnerability"
+
+	"golang.org/x/xerrors"
+)
+
+func init() {
+	log.InitLogger(false)
+	if err := db.Init(); err != nil {
+		util.Log.Debugf("library db cant initialized")
+	}
+}
+
+func FillLibrary(r *models.ScanResult) (totalCnt int, err error) {
```

Comment:
__[Golint]__ exported function FillLibrary should have comment or be unexported ([link](https://sider.review/gh/repos/54808920/pulls/829#issue-4176675))

<sub>posted by Sider</sub>